include/entity.h

```diff
@@ -2152,6 +2152,19 @@ typedef struct {
     /* 0xAD */ u8 unkAD;
 } ET_Diplocephalus;
 
+typedef struct {
+    /* 0x7C */ s32 : 32;
+    /* 0x80 */ s32 : 32;
+    /* 0x84 */ s32 : 32;
+    /* 0x88 */ s32 unk88;
+    /* 0x8C */ s16 unk8C;
+    /* 0x8E */ s16 : 16;
+    /* 0x90 */ struct Primitive* unk90;
+    /* 0x94 */ s32 : 32;
+    /* 0x98 */ s32 : 32;
+    /* 0x9C */ u8 unk9C;
+} ET_DiplocephalusFireball;
```

Comment:
Seems like there's a separate extension used here, 0x90 is a prim not an entity as it is in the other extension, similarly with 0x8C which is an s16 here but an entity in the other.